imaginary/wiring/player.py

```diff
@@ -46,7 +47,11 @@ def ebAmbiguity(err):
             err.trap(eimaginary.AmbiguousArgument)
             exc = err.value
             msg = "Could you be more specific?  When you said '" + exc.partValue + "', did you mean: "
-            format = lambda potentialTarget: ''.join(iimaginary.IConcept(potentialTarget).vt102(self.player))
+            def format(potentialTarget):
+                concept = iimaginary.IConcept(potentialTarget)
+                vt102 = concept.vt102(self.player)
+                flattened = list(flatten(vt102, useColors=False))
```

Comment:
Hmm. Not sure what purpose this serves, particularly the `flatten` call, but it probably doesn't belong in this branch...